pharus/version.py

```diff
@@ -1,2 +1,2 @@
 """Package metadata."""
-__version__ = "0.4.1"
+__version__ = "0.4.2"
```

Comment:
This component adds enough value to justify a minor release. :muscle: 

```suggestion
__version__ = "0.5.0"
```